Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -718,23 +753,57 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
         extraInfoHolder.length = tofInt.getL();
```

Comment:
we discussed during the Monday meeting the way TOF info is stored, at the moment please store the `extraInfoHolder.tofExpMom` as in run2 AOD conversion:
https://github.com/alisw/AliPhysics/blob/820236aa6f0149a27d5204b64bb52620772df90f/RUN3/AliAnalysisTaskAO2Dconverter.cxx#L1657-L1665

Corresponding methods were implemented in https://github.com/AliceO2Group/AliceO2/blob/3d7bf78847055e5e16f41b8ddb88cc60cdd36a71/Analysis/DataModel/include/AnalysisDataModel/PID/PIDTOF.h#L58-L61 but I am not sure about the template parameters, hope @njacazio can clarify what should be used

Reply:
Thank you for the snippet. I will try adding this column